From c16fad05e5c4d5036c4c12deec712f888a1d5618 Mon Sep 17 00:00:00 2001
From: Scott Ellis <scott@jumpnowtek.com>
Date: Wed, 31 Dec 2014 08:31:39 -0500
Subject: [PATCH] omap4: Do warm reset on reboot

The Duovero hangs on cold reboots.
The PandaBoard does not seem to care.
---
 sys/arm/ti/omap4/omap4_prcm_clks.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sys/arm/ti/omap4/omap4_prcm_clks.c b/sys/arm/ti/omap4/omap4_prcm_clks.c
index ec761ac..82174a8 100644
--- a/sys/arm/ti/omap4/omap4_prcm_clks.c
+++ b/sys/arm/ti/omap4/omap4_prcm_clks.c
@@ -1589,7 +1589,7 @@ omap4_prcm_reset(void)
 	reg = bus_read_4(mem_res, PRM_RSTCTRL_OFFSET);
 
 	bus_write_4(mem_res, PRM_RSTCTRL_OFFSET,
-		reg | PRM_RSTCTRL_RST_GLOBAL_COLD);
+		reg | PRM_RSTCTRL_RST_GLOBAL_WARM);
 
 	bus_read_4(mem_res, PRM_RSTCTRL_OFFSET);
 }
-- 
2.1.2

